BurnerB/FreeMentors-V2
# #168398670 Get specific mentor

#### What does this PR do?
Allows a user to view a specific  mentor
#### Description of Task to be completed?
* Create the get specific mentor mentor endpoint
* Ensure tests pass
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ft-DB-specific-mentor-168398670 
* Run npm --init to install all dependancies.
* Run command `npm test` on your terminal
* Tests should pass
* Run the command `npm start` and access GET`/api/v1/mentors/2` :
* You should recieve a response body if the data  is present:
```
"status": 200,
    "data": [
        {
            "id": 2,
            "firstname": "John",
            "lastname": "Doe",
            "email": "johndoe6@email.com",
            "address": "Nairobi Kenya",
            "bio": "rapper, record producer, and actor who was known as one of the most-controversial and best-selling artists of the early 21st century",
            "occupation": "Musician",
            "expertise": "rapping",
            "ismentor": true,
            "isadmin": false,
            "created_on": "2019-09-10T03:25:20.407Z"
        }
    ]
```
or
```
{
    "status": 404,
    "error": "No Mentor with that ID found"
}
```
if the mentor doesn't exist
#### What are the relevant pivotal tracker stories?
[#168398670](https://www.pivotaltracker.com/story/show/168398670)
#### Screenshots (if appropriate) 

![getSpecificMentor](https://user-images.githubusercontent.com/45785786/64600337-95fd4e00-d3c3-11e9-9f67-8971326be4bf.png)
